### PR TITLE
library

### PR DESCRIPTION
**Pull request by vidhishweta01**
this is a small Library app where user can keep the books which is finished reading or still to read

### CODE REVIEW

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUESTED 🔴 ♻️ 
Hello, @vidhishweta01 👋🏾 👋🏾 congratulations on making it this far 😄 . 

#### Highlights

- Your app works very well, I am able to add a new book and mark it as read ✔️ 

#### To improve

- [x] There is a small bug in your app, I am able to create a blank book. Add some form of validation so this doesn't happen
- [x] Add eslint configuration as well. You can find them [here](https://github.com/microverseinc/linters-config/tree/master/javascript)

#### Optional

- [x] In the project requirements this is marked as optional but it would be nice to persist the data to local storage. This way a user will not loose data when they refresh the page

That's all 😄, kindly submit another review when done
If you need any clarification, kindly post your comment here and tag me @mimipeshy and will be glad to help

Regards,
Peris Ndanu

**Comment by vidhishweta01** [comment]
made changes as per code review:
1. Added eslint configuration
2. Added validation against blank book
3. Added local storage

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS : CHANGES REQUIRED  ♻️ 
@vidhishweta01  Weldone, kindly implement the suggested changes from the previous Code Reviewer,  however changes is requested
### improve

- [x]  There is a small bug in your app, I am not able to create a book.
- [x] Add eslint configuration as well. You can find them [here](https://github.com/microverseinc/linters-config/tree/master/javascript)

**Comment by vidhishweta01** [comment]
eslint is already configured 

Screenshot from 2021-06-17 07-26-43

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- I can create a book :+1: 

To improve :factory_worker: [REQUIRED]:

- [ ] Please, do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
for example, in your config you have: 
`npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator`
In the original we have:
`npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x`

- [ ]  There is a small bug in your app, I am able to create book without page number. Add some form of validation so this doesn't happen


Nice to have [OPTIONAL] :

- [ ] Do not use alert() or confirm(). It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/alert_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/javascript-module-v-10/166782/path/step/97634928/))

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: scripts/main.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  if (document.getElementById('Notyetread').checked) {
+    read = document.getElementById('Notyetread').value;
+  } else {
+    read = document.getElementById('Finishedreading').value;
+  }
+
+  if (title !== '' && author !== '' && !isNaN(pages) && read !== '') { //eslint-disable-line
+    addBookToLibrary(title, author, pages, read);
+  } else {
+    document.getElementById('id01').style.display = 'block';
```
Comment (rindrajosia) [inline scripts/main.js:87]: [OPTIONAL] Keep all the styling logic in CSS files

**Review by rindrajosia — APPROVED** [review]
Status: Project Approved :green_circle:
Bravo :clap: :clap:
Great job on this project!

Here's an optional requirement if you want to take it further :arrow_double_down: :blush:

- [ ] Keep all the styling logic in CSS files (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)

With all my respect :pray:
Rindra Josia

**Pull request merged**